DataAccess/DataAccess.Test/PostgreRepositoriesContainerTest.cpp

```diff
@@ -1,6 +1,7 @@
 #include <catch2/catch.hpp>
 
 #include "DataAccess.Postgre/PostgreRepositoryContainer.hpp"
+#include "DataBaseOptions.hpp"
```

Comment:
Why this should be included?

Reply:
If you look at line 13, the namespace from this header is used.

Reply:
Yeah, but I mean that if those hardcode params were used without this, why do we need to reinclude?

Reply:
Reinclude, where? 
I guess, this is the .cpp with the tests. Perhaps in the future there will be other functionality that will require the inclusion of this header.

Reply:
If this file already uses stuff from .hpp, because this .hpp was included somewhere else, but this somewhere else is included there and we still can use it
I mean this situation